src/libs/actions/IOU.ts

```diff
@@ -863,6 +873,7 @@ function getMoneyRequestInformation(
           }
         : {};
 
+    const optimisticNextStep = NextStepUtils.buildNextStep(iouReport, CONST.REPORT.STATUS_NUM.OPEN);
```

Comment:
The report can be created as open or submitted already, we need to reflect that.

```suggestion
    const optimisticNextStep = NextStepUtils.buildNextStep(iouReport, needsToBeManuallySubmitted ? CONST.REPORT.STATUS_NUM.OPEN : CONST.REPORT.STATUS_NUM.SUBMITTED);
```

Reply:
Done - e70ec81.